routes.php

```diff
@@ -11,6 +11,7 @@
 $router->add('/agency-requests', 'agency-requests/agency_requests.php', 'Agency Requests');
 $router->add('/participant-search', 'agency-requests/agency_requests_results.php', 'Search Results');
 $router->add('/view-participant', 'agency-requests/view_participant.php', 'View Participant');
+$router->add('/live-search', 'agency-requests/live_search_test.php', 'Live Search');
```

Comment:
Is this your test file? I don't see `/live-search` referenced anywhere else but here